cabal.project

```diff
@@ -252,33 +291,49 @@ allow-newer:
     hjsonschema:*
   , hjsonpointer:*
   , cardano-sl-x509:ip
-  , cardano-addresses:aeson
+  , *:aeson
   , size-based:template-haskell
 
+-- Copied from (copied from cardano-node/cabal.project)
+allow-newer:
+  *:aeson,
+  monoidal-containers:aeson,
+  size-based:template-haskell
+
+allow-newer:
+  async-timer:unliftio-core
+
 constraints:
-    hedgehog >= 1.0.2
-  -- QUICK FIX: hedgehog >= 1.1 defines Rec constructor, PlutusIR also defines
-  -- Rec constructor, and imports Hedgehog unqualified, leading to ambiguous
-  -- reference to Rec.
-  -- FIXME: https://input-output.atlassian.net/browse/ADP-1545
-  , hedgehog < 1.1
-  , hashable < 1.4
+    hashable < 1.4
   , bimap >= 0.4.0
+  , openapi3 >= 3.2.0
   , libsystemd-journal >= 1.4.4
   , systemd >= 2.3.0
   -- dependency of systemd-2.3.0
   , network >= 3.1.1.1
   -- choose versions that work with base >= 4.12
   , hjsonpointer >= 1.5.0
   , hjsonschema >= 1.10.0
+  , Cabal >= 3.4.0.0
+  , async-timer >= 0.2.0.0
+  , unliftio-core >= 0.2.0.1
+
+  -- This pin is just here to make cabalProejct == stackProject.
+  -- Remove it if you are happy for a newer version to be picked when possible.
+  , semigroups == 0.19.2
+  , persistent < 2.13.3.4
 
 -- ----------------------------------------------------------------
 -- Flags for dependencies
 
 package cardano-config
-  flags: -systemd
+  flags: +systemd
 package cardano-node
-  flags: -systemd
+  flags: +systemd
```

Comment:
🧐 Not sure where this came from nor the implications of it. Do we want it?

Reply:
I see the node repo has docs which says

> We recommend [systemd](https://www.freedesktop.org/wiki/Software/systemd/) on Linux

and has  `systemd = !pkgs.stdenv.hostPlatform.isMusl` in its nix config.

With this change we have `flags: +systemd` but which can be overridden to [false if isMusl](https://github.com/input-output-hk/cardano-wallet/blob/1f8742817b9c7a2ab4a819d3ed284931880c371d/nix/haskell.nix#L407-L409), which should be equivalent to the node's `= !isMusl`.

So while I don't grasp the implications, I'd guess this is fine.